spec/acceptance/httpboot_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper_acceptance'
+
+describe 'Scenario: install foreman-proxy with HTTPBoot' do
+  include_examples 'the example', 'httpboot.pp'
+
+  describe service('foreman-proxy') do
+    it { is_expected.to be_enabled }
+    it { is_expected.to be_running }
+  end
+
+  describe port(8000) do
+    it { is_expected.to be_listening }
+  end
+
+  describe port(8443) do
+    it { is_expected.to be_listening }
+  end
+
+  describe command('curl --silent --write-out \'%{http_code}\' http://localhost:8000/EFI/grub2/shim.efi') do
```

Comment:
This fails because for some reason curl also prints the full response in a pipe. At least on EL8.